.github/pull_request_template.md

```diff
@@ -0,0 +1,10 @@
+---
+
+<!-- Text above this line will be added to the commit once the PR is merged -->
+
+### Pull Request checklist
+
+<!-- Before submitting the PR, please address each item -->
+
```

Comment:
Let's allocate some space in the template for where the author should place the old screenshot and new screenshot. Otherwise it won't be consistent between PRs.